### PR TITLE
feat: Handle Partytown config

### PR DESCRIPTION
**Pull request by igorbrasileiro**
### Context
The partytown plugin need to proxy variables from window, like gtag, dataLayer, fb, jitsu.

### Solution
Partytown plugin receives PartytownConfig parameter

#### Changes
Remove the ga plugin proxy

### CODE REVIEW

File: plugins/ga.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export default function ga(options: GAOptions): Plugin {
   const main = `data:application/javascript,export default function(state){
       var ga = document.createElement("script");
       ga.type = "text/partytown";
-      ga.src = "/live/proxy/gtag/js?id=${trackingId}";
+      ga.src = "https://www.googletagmanager.com/gtag/js?id=${trackingId}";
```
Comment (lucis) [inline plugins/ga.ts:12]: Just to clarify: this was only here because @guifromrio was using Brave and it had blocked GTM, right?

**Review by lucis — APPROVED** [review]

File: plugins/ga.ts (hunk 1 above)
Comment (igorbrasileiro) [reply plugins/ga.ts:12]: Yeah, probably. I had this issue, then I disabled the brave `Shield UP` and worked fined, also at chrome it works normally

**Comment by guifromrio** [comment]
Isnt it best to let it working for brave users?

Sent via Superhuman ***@***.***>

**Comment by igorbrasileiro** [comment]
> Isnt it best to let it working for brave users? Sent via Superhuman ***@***.***>

I don't have an opinion about it, but I think the brave is less unrepresentative.  https://gs.statcounter.com/browser-market-share

**Comment by guifromrio** [comment]
OK!

**Review by guifromrio — APPROVED** [review]

**Pull request merged**